### ISSUE TITLE
Trying to get the magnetic longitude for 6.63h MLT on 2011-10-06 12:46 UT gives RuntimeError

### ISSUE THREAD

**Issue by johncoxon** [cmeeren/aacgmv2#3]
```
In [296]: aa.convert_mlt(6.63, dt.datetime(2011,10,6,12,45), m2a = True)
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-296-911d07a9749d> in <module>()
----> 1 aa.convert_mlt(6.63, dt.datetime(2011,10,6,12,45), m2a = True)

/Users/John/anaconda/lib/python2.7/site-packages/aacgmv2/wrapper.pyc in convert_mlt(arr, datetime, m2a)
    185 
    186     # convert subsolar coordinates at 30Re altitude to AACGM-v2 (tracing)
--> 187     _, mlon_subsol = convert(subsol_lat, subsol_lon, 30*6371.2, datetime, trace=True)
    188 
    189     # convert the input array

/Users/John/anaconda/lib/python2.7/site-packages/aacgmv2/wrapper.pyc in convert(lat, lon, alt, date, a2g, trace, allowtrace, badidea, geocentric)
    112 
    113     # convert
--> 114     lat_out, lon_out, _ = aacgmConvert_vectorized(lat, lon, alt, flag)
    115 
    116     return lat_out, lon_out

/Users/John/anaconda/lib/python2.7/site-packages/numpy/lib/function_base.pyc in __call__(self, *args, **kwargs)
   1809             vargs.extend([kwargs[_n] for _n in names])
   1810 
-> 1811         return self._vectorize_call(func=func, args=vargs)
   1812 
   1813     def _get_ufunc_and_otypes(self, func, args):

/Users/John/anaconda/lib/python2.7/site-packages/numpy/lib/function_base.pyc in _vectorize_call(self, func, args)
   1872             _res = func()
   1873         else:
-> 1874             ufunc, otypes = self._get_ufunc_and_otypes(func=func, args=args)
   1875 
   1876             # Convert args to object arrays first

/Users/John/anaconda/lib/python2.7/site-packages/numpy/lib/function_base.pyc in _get_ufunc_and_otypes(self, func, args)
   1834             # arrays (the input values are not checked to ensure this)
   1835             inputs = [asarray(_a).flat[0] for _a in args]
-> 1836             outputs = func(*inputs)
   1837 
   1838             # Performance note: profiling indicates that -- for simple

RuntimeError: AACGM_v2_Convert returned error code -1
```

Why is this occurring? I couldn't find anything about what an error code of -1 denotes.

**Comment by cmeeren** [comment]
Thank you for the report. Karl and I are aware of problems with the MLT being undefined and causing a crash in the C library. We are currently speaking to Simon Shepherd to determine the best course of action. There is no immediate fix in sight; if you can, I suggest you try apexpy instead (which uses Apex coordinates - they're almost identical to AACGMv2 at high latitudes, and are defined everywhere).

**Comment by klaundal** [comment]
I have a fix more or less ready. I'll try to get it posted tomorrow.

**Comment by cmeeren** [comment]
@klaundal, did you talk to Simon about how best to proceed concerning MLT? Or is this the fix you originally submitted a while back?

**Comment by klaundal** [comment]
@cmeeren, This is the fix that I submitted before. I'll try to get it right this time. It's not done the way Simon suggested, but I believe this is more appropriate. We should just be careful to add documentation to explain how we do it, and that it is different from the latest C/IDL code implementation.

**Comment by cmeeren** [comment]
Yep. Be sure to update the documentation too. And after submitting the pull request, the test values probably need to be updated before merging (we'll get back to that).

**Comment by johncoxon** [comment]
@cmeeren Thanks, I've downloaded apexpy and the code falls back to using that instead when it catches a `RuntimeError`. I'll keep an eye on this library and upgrade when the problem's fixed.

**Comment by cmeeren** [comment]
Should be fixed in the brand new v2.0.0.
